components/Article.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export default function StyledArticle(props) {
+  return <Article>{props.children}</Article>;
+}
+
+const Article = styled.article`
+  text-align: justify;
+  text-align-last: none;
+  color: #572887;
```

Comment:
you can write your color ex in a variable that recall the color ( like --skyBlue)